core/src/main/scala/cats/MonoidK.scala

```diff
@@ -67,8 +84,54 @@ import simulacrum.typeclass
     new ComposedMonoidK[F, G] {
       val F: MonoidK[F] = self
     }
+
+  /**
+   * Return `a` combined with itself `n` times.
+   *
+   * Example:
+   * {{{
+   * scala> SemigroupK[List].combineNK(List(1), 5)
+   * res0: List[Int] = List(1, 1, 1, 1, 1)
+
+   * scala> MonoidK[List].combineNK(List("ha"), 0)
+   * res1: List[String] = List()
+   *
+   * }}}
+   */
+  override def combineNK[A](a: F[A], n: Int): F[A] =
+    if (n < 0) throw new IllegalArgumentException("Repeated combining for monoids must have n >= 0")
```

Comment:
I think we should say `MonoidK` not `monoid` here.

Reply:
Sorry! Good spot!

Reply:
This hasn't been fixed.